DS13/code/modules/overmap/transporter.dm

```diff
@@ -84,6 +97,25 @@
 /datum/action/innate/changearea/Activate()
 	console.choose_area(console.operator)
 
+/datum/action/innate/transporterlock
+	name = "Lock on"
+	icon_icon = 'DS13/icons/actions/actions_transporter.dmi'
+	button_icon_state = "lock"
+	var/obj/machinery/computer/camera_advanced/transporter_control/console
```

Comment:
What's this declaration good for?

Reply:
This is used to reference who is using the console for actions like beamup() and such